proto/rill/local/v1/api.proto

```diff
@@ -108,3 +111,15 @@ message GetCurrentProjectResponse {
   string local_project_name = 1;
   admin.v1.Project project = 2;
 }
+
+message GetUserOrgMetadataRequest {}
+
+message GetUserOrgMetadataResponse {
+  message OrgMetadata {
+    string name = 1;
+    bool is_admin = 2;
```

Comment:
this is not populated in the API ? is this used ?

Reply:
No I removed it in further iterations.